lucene/core/src/java/org/apache/lucene/search/Sort.java

```diff
@@ -22,80 +22,7 @@
 
 
 /**
- * Encapsulates sort criteria for returned hits.
- *
- * <p>The fields used to determine sort order must be carefully chosen.
```

Comment:
Wow, these docs were quite stale!  They hark from the `FieldCache` days.  Wow, 2004!

I think instead of fully deleting them, we should just updating them to state that you index the corresponding typed doc values field, and then sort by that type?

Reply:
++